.github/workflows/update-kernels.yml

```diff
@@ -3,12 +3,35 @@ name: Update Kernels
 
 on:
   workflow_dispatch:
+    inputs:
+      distro:
+        description: distro of which to perform the update
+        type: choice
+        required: false
+        options:
+          - '*'
+          - AmazonLinux
+          - AmazonLinux2
+          - AmazonLinux2022
+          - AmazonLinux2023
+          - BottleRocket
+          - CentOS
+          - Debian
+          - Fedora
+          - Flatcar
+          - Minikube
+          - OracleLinux
+          - PhotonOS
+          - Redhat
+          - Ubuntu
   schedule:
     - cron: '30 6 * * 1'
 
 jobs:
   update-kernels:
     runs-on: ubuntu-latest
+    env:
```

Comment:
I think we can directly use `inputs.distro` anywhere, instead of routing through an env!